From 2b0ed30806b34315962da382cb41edf5f19b231e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maciej=20Kr=C3=BCger?= <mkg20001@gmail.com>
Date: Sat, 25 Nov 2023 14:22:24 +0100
Subject: [PATCH] Add installation to PREFIX on mac when set

This allows the app to be shipped in a non-bundeled version

We need this to ship the app on macOS with nix
---
 QLog.pro | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/QLog.pro b/QLog.pro
index db6686f..576bfe1 100644
--- a/QLog.pro
+++ b/QLog.pro
@@ -386,6 +386,12 @@ macx: {
    equals(QT_MAJOR_VERSION, 6): LIBS += -lqt6keychain
    equals(QT_MAJOR_VERSION, 5): LIBS += -lqt5keychain
    DISTFILES +=
+
+   # This allows the app to be shipped in a non-bundeled version
+   !isEmpty(PREFIX) {
+     target.path = $$PREFIX
+     INSTALLS += target
+   }
 }
 
 win32: {
-- 
2.42.0

